BigQuery.Linq.Tests/Builder/StandardQueries.cs

```diff
@@ -505,13 +505,6 @@ public void EnumAsValue()
         [TestMethod]
         public void DateTime()
         {
-            var jpCalendar = CultureInfo.GetCultureInfo("ja-jp").Calendar; // +9
-            new BigQueryContext().Select(() => new { dt = new DateTime(2014, 10, 17, 6, 0, 0, jpCalendar) })
-                .ToString()
-                .Is(@"
-SELECT
-  '2014-10-16 21:00:00.000000' AS [dt]".TrimSmart());
-
```

Comment:
I see the point of the test now, only issue is that offset can change based on the date (for example, daylight savings in the UK). Made a couple of small changes:
```cs
var local = new DateTime(2014, 10, 16, 21, 0, 0, DateTimeKind.Local);
            var offset = TimeZone.CurrentTimeZone.GetUtcOffset(local);
            var offsetLocal = new DateTime(local.Ticks + offset.Ticks, DateTimeKind.Local);
            new BigQueryContext().Select(() => new { dt = offsetLocal })
                            .ToString()
                            .Is(@"
SELECT
  '2014-10-16 21:00:00.000000' AS [dt]".TrimSmart());
```